cross_win64.txt

```diff
@@ -6,7 +6,7 @@ c = 'x86_64-w64-mingw32-gcc'
 cpp = 'x86_64-w64-mingw32-g++'
 ar = 'x86_64-w64-mingw32-ar'
 strip = 'x86_64-w64-mingw32-strip'
-pkg-config = 'x86_64-w64-mingw32-pkg-config'
+pkgconfig = 'x86_64-w64-mingw32-pkg-config'
```

Comment:
Seems unrelated to this PR

Reply:
Same as above; the build system would only check for 32-bit versions of packages without the revert.

Reply:
Same as the other one; updated meson by switching to pip version. Undid revert.